CONTRIBUTE.md

```diff
@@ -0,0 +1,101 @@
+# Contribution Guideline
+
+<br>
+
+## How do I make a contribution?
+
+<br>
+
+Never made an open source contribution before? Wondering how contributions work in the in our project? Here's a quick rundown!
+
+<br>
+
+## **Here's a quick rundown on how to make a contribution to my project:**
+
+---
+
+<br>
+
+1. Find an issue that you are interested in addressing or a feature that you would like to add.
+
+   <br>
+
+2. Fork the repository associated with the issue to your local GitHub account. This means that you will have a copy of the repository `underyour-GitHub-username/repository-name.`
+
+   <br>
+
+3. Clone the repository to your local machine using:
+
+   ```
+   git clone https://github.com/github-username/repository-name.git.
+   ```
+
+4. Add the upstream remote:
+
+   ```
+    git remote add upstream https://github.com/abitsalihu/quizafriend.git
+   ```
+
+5. Pull the latest changes from the main repository if you think your fork is behind:
+
+   ```
+   git pull upstream main
+   ```
+
+6. Next install npm and run the script to start the live-server.
+
+   ```
+   npm install
+   ```
+
```

Comment:
Could you merge these two code blocks into one

Reply:
Done.